terraform/eks_fargate_setup/main.tf

```diff
@@ -0,0 +1,695 @@
+# ------------------------------------------------------------------------
```

Comment:
Would recommend using the terraform-aws-modules for EKS. It handles a lot of the weirdness for setting up EKS and I've had good experience with it

https://github.com/terraform-aws-modules/terraform-aws-eks

Reply:
Now I'm using the eks module.